### DISCUSSION TITLE
如何配置暗黑主题？

### DISCUSSION THREAD

**Discussion by Gemini-0529** [didi/LogicFlow#1650] [Q&A]
目前系统是有暗黑配色的，但是logicFlow一直是白色的，我想配置暗黑主题色，请问有什么方法？

**Comment by EdwinOlders11** [comment]
你其实把背景图一换，再把每个节点的样式改一改就好了，比如我目前在搞的一个就是：
![屏幕截图 2024-11-07 114257](https://github.com/user-attachments/assets/98c54e3c-f148-4576-b470-8075a8934fce)
